releasenotes/notes/deprecate-span-tracer-master-6c8666cf49ee4f4a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    ``Span.tracer`` is deprecated and will be removed in v1.0.
```

Comment:
Should we add a note here on what action items to take?

e.g. "if you have Span(tracer, "name", etc) then use Span(None, "name", etc) instead" 

I am not sure the right wording/action items we should suggest, but this + the `removed_property` warning should include steps since it does alter the constructor arguments

Reply:
I added `Avoid calling `Span.tracer`. Use ``Span(tracer=None, name='spanname')`` to initialize a Span object."` to the release note and deprecation warning.